metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/Context.java

```diff
@@ -36,6 +36,7 @@
     , STELLAR_CONFIG
     , CONSOLE
     , SHELL_VARIABLES
+    , CACHE
```

Comment:
If you're so inclined, it would be useful to add some javadoc for `CACHE`.  I know we haven't done it on the others, but we should.

Reply:
I can remember from experience trying to track down what some of these are used for in the past.  A little doc here would have been very helpful.